ios/app/src/Info.plist

```diff
@@ -55,8 +55,8 @@
 			</dict>
 		</dict>
 	</dict>
-        <key>NSCalendarsUsageDescription</key>
-        <string>See your scheduled conferences in the app.</string>
+    <key>NSCalendarsUsageDescription</key>
+    <string>See your scheduled conferences in the app.</string>
```

Comment:
I don't understand... Even after the modification, the keys are not aligned like all the other keys in the file!

Reply:
Weird indeed, because I remember I checked it when I originally added the feature too and still had to re-align. Maybe something modifies it on native build(?)